libkbfs/cr_chains.go

```diff
@@ -799,3 +802,65 @@ func (ccs *crChains) changeOriginal(oldOriginal BlockPointer,
 	}
 	return nil
 }
+
+// getPaths returns a sorted slice of most recent paths to all the
+// nodes in the given CR chains that were directly modified during a
+// branch, and which existed at both the start and the end of the
+// branch.  This represents the paths that need to be checked for
+// conflicts.  The paths are sorted by descending path length.  It
+// uses nodeCache when looking up paths, which must at least contain
+// the most recent root node of the branch.  Note that if a path
+// cannot be found, the corresponding chain is completely removed from
+// the set of CR chains.
+func (ccs *crChains) getPaths(ctx context.Context, blocks *folderBlockOps,
```

Comment:
Sorry for the ignorance, but I'm confused why edit history requires a conflict resolution chain. Does a CR chain get created for any set of edits?